### ISSUE TITLE
X की सेवा/देखभाल

### ISSUE THREAD

**Issue by aryamanarora** [aryamanarora/carmls-hi#18]
* X **की** सेवा/देखभाल
_X **GEN** service/care_
_care/service for X_

Beneficiary↝Theme? Just Theme?

**Comment by nitinvwaran** [comment]
I'm going to propose something different, which is **Gestalt**.  Based on the following sentences from Sketch Engine:

> यही के कोई घर पर उसकी देखभाल के लिए था
> someone home at his care for was (where for = के लिए, someone = यही के कोई) 
> The के लिए loosely translates to for. So it might be that  उसकी देखभाल is loosely 'his care'.

> और हमारी सेवा करेगा या नहीं ओल्ड ऐज में
> and our service do.3SG.FUT or no old age in 
> The करेगा verb takes as its complement, हमारी सेवा, which might suggest a noun-like element, maybe loosely, 'our care'. 

There are lots of examples like the second one in Sketch Engine. Also, 

> RSS देश के सेवा के लिए ही बनाई गई है
> RSS country GEN service for only made is 
> RSS is only established for the country's service

Therefore Gestalt, because of unalienable properties associated with the X, like service, care, and maybe even help i.e देखभाल, सेवा, and मदद resp.

**Comment by aryamanarora** [comment]
Hmm. The problem with this is that सेवा करना occurs as a compound verb construction too (which are messy in themselves, see #14). सेवा is an action, so I am wary of making it just plain Gestalt, since that tends to cover abstract possession rather than relations between a participant and an action. And सेवा is, I don't think, a property.

For example:

> मैंने उस**की** सेवा की।
> 1S.ERG 3S.**GEN** service do.PFV
> I served him/I cared for him.

The English translation really highlights that the scene role of उस is Theme, because something is being done to it.

> उस**की** पिटाई

Similarly, something being done to the object, so it's Theme.

Actually, looking back this may just be a plain Theme; I want to rule Beneficiary~Theme out for sure. You mentioned "help" as something similar, and I found example 202d in the English guidelines: they needed Joan'**s** help (Agent\~Gestalt). The counterpart of a helper (Agent) would be a Theme in that case. I'm not totally sure on Theme though still.

**Comment by nitinvwaran** [comment]
Yeah, on the Sketch Engine Hindi Dataset both सेवा and मदद occur mostly as part of a complex verb form, so i dont think it can be a property of anything.

_Modern Hindi Grammar_ Omkar N Koul (2009),  I think confirms Theme (page 56, uses of ka, 'Object of an activity'). Examples from his book:

> उसके बच्चों की शिक्षा अच्छी हे
> उसके पास दवाई का खर्चा नहीं हे 

His second example is really interesting. I think the Parts of Speech here are (Noun का Noun) i.e (दवाई का खर्चा) or maybe literally, 'expenditure of medicines', but the दवाई is framed as an Object of an activity! Maybe the activity is to expend on medicines. 

I think for our purposes, if we frame सेवा/देखभाल as activities, then X becomes Theme in line with Koul (2009). 
But based on the दवाई का खर्चा example, दवाई is clearly not benefitting from the Scene. 
बच्चों की शिक्षा is also an example of where बच्चों is not benefitting however the बच्चों is in a Beneficiary Scene in the full phrase: बच्चों की शिक्षा अच्छी हे.

To conclude i think X की सेवा/देखभाल may not be enough to determine Scene, and in the absence of any other adjectival / adverbial information i would probably mark it pure Theme~Theme.
